components/brave_rewards/resources/extension/brave_rewards/content.ts

```diff
@@ -0,0 +1,145 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This script is pretty site-specific, should we name it accordingly?

Reply:
Yes, good idea.